index.html

```diff
@@ -162,7 +95,7 @@ <h4>Thank you for your interest!</h4>
       </li>
     </ul>
 
-    <div id="right-div">
+    <div id="right-div" class="valign-wrapper">
```

Comment:
Is this a typo "valign"?

Reply:
It's a materialize property. Stands for "Vertical align"